ee/clickhouse/models/event.py

```diff
@@ -59,28 +61,24 @@ def create_event(
 
     p.produce_proto(sql=INSERT_EVENT_SQL, topic=KAFKA_EVENTS, data=pb_event)
 
-    if team.slack_incoming_webhook or team.organization.is_feature_available("zapier"):
-        # Do a little bit of pre-filtering
-        if event in ActionStep.objects.filter(action__team_id=team.pk, action__post_to_slack=True).values_list(
-            "event", flat=True
-        ):
-            try:
-                celery.current_app.send_task(
-                    "ee.tasks.webhooks_ee.post_event_to_webhook_ee",
-                    (
-                        {
-                            "event": event,
-                            "properties": properties,
-                            "distinct_id": distinct_id,
-                            "timestamp": timestamp,
-                            "elements_list": elements,
-                        },
-                        team.pk,
-                        site_url,
-                    ),
-                )
-            except:
-                pass
+    if Action.objects.exists():
```

Comment:
This introduces the same problem as the one being fixed - it does a query against the database for every event we receive. 

We should cache this between events.

Reply:
I see, yeah, a way out of this then: caching `Organization.available_features` in Redis with a TTL of, say, 60 s?

Reply:
That works for now.

In an ideal world, we shouldn't hit DB/cache for every event though as it introduces a bottleneck. WH could work off of WAL (or similar) as well and keep internal cache for relevant objects which they refresh every 30s. Not in scope for this PR though.

Reply:
I added in-memory cache in Python – the memory impact should be negligible considering that the set of organizations is relatively small and very constant.